.gitignore

```diff
@@ -1,3 +1,2 @@
-/.idea/
-/composer.lock
```

Comment:
My thinking behind this was that the composer.lock file is ignored by composer anyway, when using this package as a library. This was just that all developers, working on this library, work with the same versions.

Reply:
Yes, but since you are unable to control with which versions of dependencies your library is installed, you might as well impose this restriction on all developers.